src/js/table.js

```diff
@@ -0,0 +1,235 @@
+(function (Wood) {
+    Wood.Table = Marionette.ItemView.extend({
```

Comment:
we should talk a bit how to make this more generic -- this is going to be pretty specific to ORB responses right now I think